_literate/EX-powergen/tutorial.jl

```diff
@@ -17,8 +17,6 @@ end;
 
 using MLJ
 using UrlDownload
-using PyPlot
```

Comment:
Can you please check you have a `using Plots` somewhere here?

Reply:
Duplicate.